ui/src/views/WorkerManager/routes.js

```diff
@@ -24,7 +24,8 @@ export default path => [
   },
   {
     component: WMWorkerPoolEditor,
-    path: `${path}/:workerPool/edit`,
+    isNewWorkerPool: false,
+    path: `${path}/:workerPoolId/edit`,
     description: 'Edit the worker pool',
   },
   {
```

Comment:
I never noticed this - for me, it was encoded in the url. I'll check this

Reply:
oh, so that route is for the workers view, `WorkerManagerViewWorkers` - that component is in the works and out of scope of this PR (and it's not related to the code you commented on). Please ignore